src/components/ShowList.css

```diff
@@ -20,3 +20,9 @@
 .checked {
   text-decoration: line-through;
 }
+
+button:focus,
```

Comment:
This looks good as it is. But if you want to make it shorter, `button:hover:focus` would do the same thing

Reply:
Looks like it won't do the _exact_ same thing, I think they need to be separated like they are.

Reply:
Ah okay, I tested it incorrectly then. I’ll resolve it. Thanks kevin!